SU-SWS/decanter
# resolve 184: stop importing .css file into .scss

# READY FOR REVIEW

# Summary
- TL;DR stop importing `normalize.css` into our `.scss`, as that functionality is deprecated

# Needed By (Date)
- Before webpack stops supporting importing `.css` files

# Urgency
- Low

# Steps to Test
1. `npm install` to remove the dependency on `normalize.css`
1. `grunt styleguide` to rebuild our .css and the styleguide
1. Verify that nothing has changed. It may be easiest to confirm that the first 143 lines of the new `decanter.css` are the same as the first 299 lines of the old one (removed all block style comments for CodeClimate).

# Affected Projects or Products
- N/A

# Associated Issues and/or People
- #184 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
